internal/controller/ebpf/agent_controller.go

```diff
@@ -761,7 +761,7 @@ func getEnvConfig(coll *flowslatest.FlowCollector, cinfo *cluster.Info) []corev1
 	})
 
 	defaultAttach := "tcx"
-	if old, _ := cinfo.IsOpenShiftVersionLessThan("4.15.0"); old {
+	if old, _ := cinfo.IsOpenShiftVersionLessThan("4.16.0"); old {
```

Comment:
I am curious how did we use to fallback on `tc` hook prior to https://github.com/netobserv/network-observability-operator/pull/1670 for older versions?  

I am not sure if there's a way to check a RHEL kernel version instead which would make it more explicit.

Reply:
(we could also try to be smart and run a one-time check at startup to see if TCX is supported; my fear is just that in case it fails for any reason we're no further ahead ; having things hard-written in config makes everything more deterministic.)